.circleci/config.yml

```diff
@@ -831,6 +831,7 @@ defaults:
       project: uniswap
       binary_type: native
       image: cimg/node:18.16
+      resource_class: large
```

Comment:
If this fixes it, add a comment here indicating that the resource class was bumped because we were running into OOM issues.

Reply:
Done